docusaurus.config.js

```diff
@@ -136,23 +128,24 @@ const config = {
               label: "Ledger Explorer",
               href: "https://stellar.expert/explorer/public",
             },
-            { label: "All Tools", href: "https://www.stellar.org/tools" },
+            { label: "All Tools", 
+              href: "https://developers.stellar.org/docs/tools-and-sdks/" },
           ],
         },
         {
-          title: "Developers",
+          title: "Community",
           items: [
             {
-              label: "Developer Resources",
-              href: "https://www.stellar.org/developers-legacy",
+              label: "Stellar Developer Discord",
+              href: "https://discord.com/invite/zVYdY3ktTn",
             },
             {
-              label: "Status",
```

Comment:
We should definitely keep this link. It's one of the most important for pro tier devs. We should also include this one https://dashboard.stellar.org/

Reply:
Both of these links likely belong in tools

Reply:
I was just trying to shorten the list so the footer isn't ridiculously long. Just want to keep the top 4 most used then link to the rest of the tools. Most used are? Account Viewer, Dashboard, Ledger Explorer, Status? Or something else?